Dockerfile

```diff
@@ -1,32 +1,79 @@
-FROM node:0.12
+# Build:
+# docker build -t lirantal/meanjs .
+#
+# Run:
+# docker run -it lirantal/meanjs
 
-# Install gem sass for  grunt-contrib-sass
-RUN apt-get update -qq && apt-get install -y build-essential
+FROM ubuntu:latest
+MAINTAINER Liran Tal <liran.tal@gmail.com>
```

Comment:
We removed the maintainer line before, but I don't remember the reason why. Just want to make sure we want it put back.
https://github.com/meanjs/mean/commit/710c70da61d120505c2fbbfce2c0175205692f61